src/appshell/qml/dockwindow/DockTitleBar.qml

```diff
@@ -46,6 +46,13 @@ Item {
 
     visible: Boolean(titleBarCpp)
 
+    MouseArea {
```

Comment:
This one is not working. The problem seems to be that `KDDW.TitleBarBase` already contains another MouseArea; because the `KDDW.TitleBarBase { … }` comes after the new `MouseArea { … }`, the `KDDW.TitleBarBase` MouseArea is placed on top of the new one, and blocks the new one from having any effect.

Solution: just put the new MouseArea _after_ `KDDW.TitleBarBase { … }`.